### DISCUSSION TITLE
Character Import

### DISCUSSION THREAD

**Discussion by lcizzle** [Sixze/ALS-Refactored#470] [General]
Hello,
With the original ALSV4 I can get my custom character working no problem. Hide the AnimMan mesh and add my own with an animation blueprint to retarget pose of mesh. I guess i'm not understanding ALS Refactored totally yet. I see the regular mesh and an overlay skeletal mesh. If I set the original mesh to not visible my overlay mesh will stop animating. I've tried many combinations and the only one that works is both meshes visible. Everything works and animates correctly but that's obviously not the best end result to have 2 meshs visible. Anyone know what I am missing?

**Comment by lcizzle** [comment]
Weird must be something off with my mesh. UE4 Mannequin and UE5 Mannequin work just fine.

**Comment by ameaninglessname** [comment]
Sounds like your mesh stop tick animation when hidden. You could try to change the mesh tick option to 'AlwaysTickPoseAndRefreshBones'
